index.html

```diff
@@ -0,0 +1,193 @@
+        <li><a href="#"><img src="image/Icon-Medium.png" alt="M" class="media"></a></li>
+    </ul>
+</div>
+
+<div class="arrow">
+    <div class="bar">
+        <h5>Work</h5>
+        <img src="image/arrow.svg" alt="down-arrow"/> 
+    </div>
+    <div class="bar, bar1">
+        <img src="image/arrow.svg" alt="down-arrow"/>
+        <h5>Work</h5> 
+    </div>
+</div>
+<div>
+    <h2 class="work">MY RECENT WORK</h2>
+    <img src="images/Vector5.png" alt="bar" class="img" id="line">
+</div>
+<section class="work-section">
+    <div class="placeholder">
+        <div class="holder">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
+            
+        </div>
+        <div class="stories">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+    
+    <div class="background-image1">
+        <div class="holder1">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
+            
+        </div>
+        <div class="stories1">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image2">
+        <div class="holder">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
+            
+        </div>
+        <div class="stories">
+            <h3>Multi-Post Stories</h3>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+            <ul class="items">
+                <li>css</li>
+                <li>html</li>
+                <li>bootstrap</li>
+                <li class="ruby">Ruby</li>
+            </ul>
+            <button class="button" type="button">See Project</button>
+        </div>
+    </div>
+
+    <div class="background-image3">
+        <div class="holder1">
+            <img src="images/ImgPlaceholder.png" alt="blank-placeholder">
```

Comment:
- [ ] Kindly give this image a width: 100% and remove the padding as her width is too long and it causes a horizontal scroll.